traffic_ops/app/db/create_tables.sql

```diff
@@ -37,6 +37,30 @@ CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
 
 SET search_path = public, pg_catalog;
 
+--
+-- Name: fast_forward_schema_migrations_version(); Type: FUNCTION; Schema: public; Owner: traffic_ops
+--
+
+CREATE OR REPLACE FUNCTION fast_forward_schema_migrations_version()
+	RETURNS TRIGGER
+	LANGUAGE PLPGSQL
+AS $$
+BEGIN
+	IF EXISTS (SELECT FROM information_schema.columns WHERE table_name = 'goose_db_version') THEN
+		UPDATE schema_migrations SET "version" = (SELECT version_id FROM goose_db_version
+			ORDER BY TSTAMP DESC
+			LIMIT 1);
+		ALTER TABLE goose_db_version
+			RENAME TO goose_db_version_unused;
+	END IF;
+	DROP TRIGGER fast_forward_schema_migrations_trigger ON schema_migrations;
+	DROP FUNCTION fast_forward_schema_migrations_version();
+	RETURN NULL;
+END$$;
+
+ALTER FUNCTION public.fast_forward_schema_migrations_version OWNER TO traffic_ops;
```

Comment:
This `schema_migrations` functions should not be added to `create_tables.sql`, it should be completely removed. It only existed to migrate from goose to Migrate, and keeping the function breaks `create_tables.sql`. From `trafficops.log` within the `ciab-logs.zip` artifact of the *CDN-in-a-Box CI* workflow:

```shell
CREATE TRIGGER fast_forward_schema_migrations_trigger
	AFTER UPDATE
	ON schema_migrations
	EXECUTE PROCEDURE fast_forward_schema_migrations_version();
ERROR:  relation "schema_migrations" does not exist

ERROR:root:Failed to set up database: Database load_schema failed
ERROR:root:Can't connect to the database.  Use the script `/opt/traffic_ops/install/bin/todb_bootstrap.sh` on the db server to create it and run `postinstall` again.
Error on line 38 of /opt/traffic_ops/install/bin/postinstall
+++ echo 'Error on line 217 of /run-go.sh'
+++ exit 1
+ cleanup
+ rm -f /tmp/tmp.AzSOeC5roH /tmp/tmp.Cw8OexrngA
Error on line 217 of /run-go.sh
```

Reply:
That's probably why the database tests fail